go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/jcdea/aarch64-client-go
```

Comment:
This is the wrong package URI, it should be `github.com/natesales/aarch64-client-go`

Reply:
Oop sorry, I forgot to change the package url